.github/workflows/make_leo_pr.yml

```diff
@@ -57,7 +93,7 @@ jobs:
           github-token: ${{ secrets.BROADBOT_TOKEN }}
           script: |
             const result = await github.rest.pulls.create({
-              title: '${{ github.event.inputs.jira_ticket }}: Update Cromwell version to ${{ github.event.inputs.new_cromwell_helm_version }}',
+              title: '[AUTOMERGE] Update Cromwell version to ${NEW_CROMWELL_HELM_V}',
```

Comment:
Do we need to have ticket number associated with PRs that are auto-merged?

Reply:
great question - Chris has observed that similar tickets into terra-helmfile didn't seem to have them in their titles, so I think we're ok? 

(there's a link in the body of the ticket: https://broadworkbench.atlassian.net/browse/WM-1468)

Reply:
Sounds good 👍